src/main/java/edu/harvard/iq/dataverse/externaltools/ExternalToolHandler.java

```diff
@@ -111,6 +111,7 @@ public String handleRequest(boolean preview) {
                 case DATASET:
                     callback=SystemConfig.getDataverseSiteUrlStatic() + "/api/v1/datasets/"
                             + dataset.getId() + "/versions/:latest/toolparams/" + externalTool.getId();
+                    break;
                 case FILE:
                     callback= SystemConfig.getDataverseSiteUrlStatic() + "/api/v1/files/"
                             + dataFile.getId() + "/metadata/" + fileMetadata.getId() + "/toolparams/"
```

Comment:
I'm just noting that this newer code (inside the `else`) doesn't have any test coverage.

From https://coveralls.io/builds/60355776/source?filename=src%2Fmain%2Fjava%2Fedu%2Fharvard%2Fiq%2Fdataverse%2Fexternaltools%2FExternalToolHandler.java

![Screen Shot 2023-05-26 at 4 46 21 PM](https://github.com/IQSS/dataverse/assets/21006/0d6d1b74-f04d-411c-a96b-6764a7fc42c2)